Source/WebCore/PAL/pal/graphics/WebGPU/Impl/WebGPUSwapChainImpl.cpp

```diff
@@ -44,12 +44,11 @@ SwapChainImpl::SwapChainImpl(WGPUSurface surface, WGPUSwapChain swapChain)
 
 SwapChainImpl::~SwapChainImpl()
 {
-    wgpuSwapChainRelease(m_backing);
+    destroy();
 }
 
 void SwapChainImpl::destroy()
 {
-    wgpuSwapChainRelease(m_backing);
```

Comment:
I think it's wrong to delete all calls to `wgpuSwapChainRelease()`. I think the fact that there's only a single object under the hood shouldn't be visible from outside WebGPU.framework.

Reply:
Good idea, I'll increment the ref count instead on m_backing